src/core/filter/security.go

```diff
@@ -144,6 +147,20 @@ type ReqCtxModifier interface {
 	Modify(*beegoctx.Context) bool
 }
 
+// configCtxModifier populates to the configuration values to context, which are to be read by subsequent
+// filters.
+type configCtxModifier struct {
+}
+
+func (c *configCtxModifier) Modify(ctx *beegoctx.Context) bool {
+	m, err := config.AuthMode()
```

Comment:
the AuthMode will return a `db_auth` as default value when encountering any error. 
And the request will be treated as a `db_auth` request in following, is it as expected?

Reply:
When error happens it will return `""`
https://github.com/goharbor/harbor/blob/master/src/common/config/manager.go#L193